noobaa/noobaa-core
# [Backport to 5.8]: Remove ssh2

### Explain the changes
- Remove ssh2
- Delete node_ssh.js
- Delete ssh_functions.js
- Cleaned the code

Signed-off-by: liranmauda <liran.mauda@gmail.com>
(cherry picked from commit 302b425e0fad7d50701f1771f6a3a2e7f4fb2c82)

### Issues: Fixed #xxx / Gap #xxx
1. Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=2015209

### Testing Instructions:
1. log in to the pod and see that there is no ssh2 module there:
```
$ kubectl exec -it pod/noobaa-core-0 -- bash
[noob@noobaa-core-0 /]$ ls /root/node_modules/noobaa-core/node_modules/
```
